Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env groovy
+// vim: set ts=4 sts=4 sw=4 noet:
+
```

Comment:
We have some common Jenkins functions in the shared libraries. They can be used by adding `@Library('lco-shared-libs@0.0.10') _` here.